common/common.mak

```diff
@@ -83,8 +83,8 @@ $(LANGUAGE_NAME).pc: ../../bindings/c/$(LANGUAGE_NAME).pc.in
 		-e 's|=$(PREFIX)|=$${prefix}|' \
 		-e 's|@PREFIX@|$(PREFIX)|' $< > $@
 
-$(PARSER): $(SRC_DIR)/grammar.json
-	$(TS) generate --no-bindings $^
+$(PARSER): grammar.js
+	$(TS) generate --no-bindings
```

Comment:
Revert this.

Reply:
@ObserverOfTime Isn't it worse to allow `grammar.js` to fall out of sync than to require node be installed? It's a real dependency of the build process. For what it's worth, I spent a non-trivial amount of time trying to figure out why my changes weren't being taking effect - I generally expect make to build the product from the latest source.

Reply:
Similar to how `cargo build`, `go build` and `npm install` are compiling/building language bindings, `make` is compiling the C bindings. They're doing that under the assumption that the `parser.c` files are already generated, so they don't force `npm` and `tree-sitter` dependencies on everyone.

I think this whole rule should be removed, since it introduces a dependency on the `tree-sitter` executable.